### PR TITLE
Jaymes search snippet

### PR DESCRIPTION
**Pull request by Jaym3s**
screenshot 2016-02-18 16 13 42

### CODE REVIEW

**Comment by Jaym3s** [comment]
https://www.pivotaltracker.com/story/show/113457185

**Comment by Jaym3s** [comment]
divoxx @brupm @jeyb

File: app/decorators/article_decorator.rb [hunk 1]
```diff
@@ -41,6 +41,23 @@ def freshness
     content_tag(:span, "fresh", class: "state fresh") if source.fresh?
   end
 
+  def any_footer_content?
+    tags.any? || matched_content_snippet.present?
+  end
+
+  def matched_content_snippet
+    if from_search?
+      snippit = SearchSnippet.new(context[:search_params], source.content)
```
Comment (brupm) [inline app/decorators/article_decorator.rb]: snippet?

Comment (Jaym3s) [inline app/decorators/article_decorator.rb]: doh!

**Comment by divoxx** [comment]
:+1:

**Comment by brupm** [comment]
:+1:

**Pull request merged**